setup.py

```diff
@@ -41,7 +41,7 @@
         'shapely',
         'pymongo',
         'numpy',
-        'rasterio',
+        'rasterio==0.31.0',
```

Comment:
@aashish24 versions above that throw an exception, seems to be some kind of bug in rasterio